### ISSUE TITLE
Allow selection of storage pool when creating a new VM

### ISSUE THREAD

**Issue by JeffWDH** [cockpit-project/cockpit-machines#1479]
While creating a new VM you currently have an option to create a new volume, use no storage, or pick a pre created disk image from a storage pool. When selecting the new volume option the destination is hard coded to the "default" storage pool.  It would be great to have the ability to select from the list of available storage pools.

**Comment by strboul** [comment]
I have the same problem. As a workaround, I created the VM with "No storage" option and later added the disk from the storage pool.

**Comment by erentar** [comment]
Yes please. The current workaround is to click on the storage pools, create the volume, then use that volume in the vm creation screen, which is significantly less convenient

**Comment by mvollmer** [comment]
There is the "Create and edit" action in the "Create VM" dialog, and this is the general way to make any kind of changes to your VM that can't do in the "Create" dialog.

The idea is that instead of piling more and more features into the "Create" dialog, we can use the whole details view of the VM-to-be to configure all aspects of it.

In this particular case, you would select "No storage", click "Create and edit", and then use "Add disk" to attach the volume you want. The "Add disk" dialog can create new volumes in arbitrary pools.

Is that good enough?

**Comment by mvollmer** [comment]
Also see https://issues.redhat.com/browse/RHEL-87337
